drivers/sensor/tenstorrent/pvt/pvt_tt_bh.c

```diff
@@ -385,6 +389,27 @@ static int pvt_tt_bh_init(const struct device *dev)
 	/* Wait for all sensors to power up, TS takes 256 ip_clk cycles */
 	k_usleep(100);
 
+	/* Initialize the single thermal calibration fuse value from TS0's 25 degree data */
+
+	const struct sensor_value celcius25 = {.val1 = 25, .val2 = 0};
+
+	for (uint8_t id = 0; id < pvt_cfg->num_ts; ++id) {
+		uint32_t deg25_start = 2240 + (id * 64);
+		uint32_t deg25_end = deg25_start + 15; /* data is 16 bits */
+
+		uint16_t efuse = (uint16_t)ReadFunctionalEfuse(deg25_start, deg25_end);
+		float efuse_celcius = pvt_tt_bh_raw_to_temp(efuse);
+
+		/* Disregard the calibration value if it is more than one degree away from 25 */
+		if (efuse_celcius < 24.0f || efuse_celcius > 26.0f) {
+			pvt_data->therm_cali[id] = 0;
+		} else {
```

Comment:
Assuming the `data` instance is either static or in the global context, this can probably be skipped, since it would be zero initialized with the rest of BSS.